src/olympia/editors/tests/test_models.py

```diff
@@ -399,7 +399,7 @@ def test_authors(self):
         AddonUser.objects.create(addon=addon, user=bert)
         AddonUser.objects.create(addon=addon, user=ernie)
         row = self.Queue.objects.all()[0]
-        assert row.authors == [(bert.id, 'bert'), (ernie.id, 'ernie')]
+        assert row.authors == [(ernie.id, 'ernie'), (bert.id, 'bert')]
```

Comment:
I'm not sure the ordering will be guaranteed. Might be best to check `len(row.authors) == 2` and `set(row.authors) == set([(ernie.id, 'ernie'), (bert.id, 'bert')])`. There's a helper to do that so you can just write:

```py
self.assertSetEqual(row.authors, [(ernie.id, 'ernie'), (bert.id, 'bert')])